internal/env.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package internal
+
+import (
+	"os"
+	"strconv"
+)
+
+// BoolEnv returns the parsed boolean value of an environment variable, or
+// def if it fails to parse.
```

Comment:
```suggestion
// def otherwise.
```
I'm thinking it may also return the default if the env. var is not set, which is technically _not_ a parsing error, just an inexistent variable.

Reply:
Good point. 👍